waitforx/waitforx.c

```diff
@@ -21,20 +21,23 @@ alarm_handler(int signal_num)
      *
      * Prefix the message with a newline in case another message
      * has been partly output */
+
+    if( signal_num == SIGALRM ) 
+    {
     const char msg[] = "\n<E>Timed out waiting for RandR outputs\n";
     g_file_write(1, msg, g_strlen(msg));
     exit(XW_STATUS_TIMED_OUT);
+    }
```

Comment:
```suggestion
    {
        const char msg[] = "\n<E>Timed out waiting for RandR outputs\n";
        g_file_write(1, msg, g_strlen(msg));
        exit(XW_STATUS_TIMED_OUT);
    }
```

Reply:
Yep, I'll have to review the indentation.